R-package/README.md

```diff
@@ -14,63 +19,56 @@
 
-```r
-PKG_URL <- "https://github.com/microsoft/LightGBM/releases/download/v3.0.0/lightgbm-3.0.0-r-cran.tar.gz"
+When your package installation is done, you can check quickly if your LightGBM R-package is working by running the following:
 
-remotes::install_url(PKG_URL)
+```r
+library(lightgbm)
+data(agaricus.train, package='lightgbm')
+train <- agaricus.train
+dtrain <- lgb.Dataset(train$data, label = train$label)
+model <- lgb.cv(
+    params = list(
+        objective = "regression"
+        , metric = "l2"
+    )
+    , data = dtrain
+)
 ```
 
-### Installing Precompiled Binaries
-
-Starting with `LightGBM` 3.0.0, precompiled binaries for the R package are created for each release. These packages do not require compilation, so they will be faster and easier to install than packages that are built from source. These packages are created with R 4.0 and are not guaranteed to work with other R versions.
+### Installing the CRAN package
 
-Binaries are available for Windows, Mac, and Linux systems. They are not guaranteed to work with all variants and versions of these operating systems. Please [open an issue](https://github.com/microsoft/LightGBM/issues) if you encounter any problems.
+As of this writing, `LightGBM`'s R package is not available on CRAN. However, start with `LightGBM` 3.0.0, you can install a released source distribution. This is the same type of package that you'd install from CRAN. It does not require `CMake`, Visual Studio, or anything else outside the CRAN toolchain.
 
-To install a binary for the R package:
+To install this package on any operating system:
 
-1. Choose a release from [the "Releases" page](https://github.com/microsoft/LightGBM/releases).
-2. Choose a file based on your operating system. Right-click it and choose "copy link address".
-    * Linux: `lightgbm-{VERSION}-r40-linux.tgz`
-    * Mac: `lightgbm-{VERSION}-r40-macos.tgz`
-    * Windows: `lightgbm-{VERSION}-r40-windows.zip`
+1. Choose a release from [the "Releases" page](https://github.com/microsoft/LightGBM/releases)
+2. Look for the artifact with a name like `lightgbm-{VERSION}-r-cran.tar.gz`. Right-click it and choose "copy link address".
 3. Copy that link into `PKG_URL` in the code below and run it.
 
-This sample code installs version 3.0.0-1 of the R package on Mac.
-
 ```r
-PKG_URL <- "https://github.com/microsoft/LightGBM/releases/download/v3.0.0rc1/lightgbm-3.0.0-1-r40-macos.tgz"
-
-local_file <- paste0("lightgbm.", tools::file_ext(PKG_URL))
+PKG_URL <- "https://github.com/microsoft/LightGBM/releases/download/v3.0.0/lightgbm-3.0.0-r-cran.tar.gz"
 
-download.file(
-    url = PKG_URL
-    , destfile = local_file
-)
-install.packages(
-    pkgs = local_file
-    , type = "binary"
-    , repos = NULL
-)
+remotes::install_url(PKG_URL)
 ```
 
-### Preparation
+### Installing from Source with CMake <a name="install"></a>
```

Comment:
I've aliased this to `#install` so that old links still work

Reply:
Wow that's cool. Did not even think this is possible.

Reply:
Indeed, nice hack! But right now contents item isn't working. I believe, removing a space before `<a name ...` will fix it.

![image](https://user-images.githubusercontent.com/25141164/92406979-0c04d400-f142-11ea-8113-c2aebc0911f9.png)

Reply:
the spacing for HTML tags is irrelevant, except for very special ones like `<pre>`. The issue here was that I got the anchor wrong in the table of contents: https://github.com/microsoft/LightGBM/pull/3360/commits/df5bf3f6b193b68a9e83dc16b5e31341f142a355

this is working now

![image](https://user-images.githubusercontent.com/7608904/92428004-9d307680-f153-11ea-8ea2-c994f3cde2a3.png)